src/include/btree_cmp_inline.h

```diff
@@ -88,10 +88,18 @@ __wt_lex_compare(const WT_ITEM *user_item, const WT_ITEM *tree_item, bool prefix
     /*
      * Use the non-vectorized version for the remaining bytes and for the small key sizes.
      */
-    for (; len > 0; --len, ++userp, ++treep)
+    for (; len > 0; --len, ++userp, ++treep) {
```

Comment:
@keithbostic, I suggested for you to be included in this PR for you to review this section of code. If you disagree with that suggestion feel free to remove yourself from the reviewer list.

Reply:
yes thats correct, the length of the string doesn't change the sorting order of the strings.